dug_6_osfv.md

```diff
@@ -0,0 +1,152 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# TBD: Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
+* Exploring OSFV v0.2
+* Recent improvements
+* Getting started with OSFV
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# Current state
```

Comment:
I miss stats, how many tests we have, what are we cover?

Reply:
added stats: fdff2448bf004b884f20796ce4d58ae8c0a47eac